build-and-test-with-yarn/action.yaml

```
@@ -21,6 +61,18 @@ runs:
       working-directory: ./
       shell: bash
 
+    - name: Test
+      run: |
+        if exit | yarn run 2>&1 | grep "\- build"
```

We are looking for test, do we?
```suggestion
        if exit | yarn run 2>&1 | grep "\- test"
```